src/App.tsx

```diff
@@ -173,33 +178,46 @@ function getMessage(logging: JSON): undefined | string[][] {
               `${logs[msg]["message"]["source"]}`
             )
             const text = logs[msg]["message"]["message"];
-            const [debug_level, log_message] = logLevel(text);
+            const [debug_level, log_message,level] = debugLevel(text);
             debug.push(debug_level);
             message.push(log_message);
+            log_level.push(level);
 
           }
-          return [timestamp,source,debug,message];
+          return [timestamp,source,debug,message,log_level];
         }
       }
     }
   }
 }
 
-function logLevel(text: string): [string, string] {
-  const debug_levels = {"INFO":1,"DEBUG":2,"WARN":3,"ERROR":4};
+function debugLevel(text: string): [string, string, number] {
```

Comment:
Does this branch need a rebase?

```suggestion
function logLevel(text: string): [string, string, number] {
```